V/vegafusion/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+using BinaryBuilder
+
+name = "vegafusion"
+version = v"0.0.4"
+
+sources = [
+    GitSource("https://github.com/vegafusion/vegafusion.git",
+                  "c0927268fc7544878c58a606e01503ba3fed615e"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+export TARGET_CC=$CC
+
+cd ${WORKSPACE}/srcdir/vegafusion/vegafusion-server/
+cargo build --release
+mkdir -p "${bindir}"
+cp "../target/${rust_target}/release/vegafusion-server${exeext}" "${bindir}/."
+install_license ../LICENSE
+"""
+
+platforms = supported_platforms()
+# Our Rust toolchain for i686 Windows is unusable
+filter!(p -> !Sys.iswindows(p) || arch(p) != "i686", platforms)
+
+# Build fails with error 'BinaryBuilder: Cannot force an architecture via -march'
+# while compiling c code in https://github.com/briansmith/ring dependency
+filter!(p -> arch(p) ∉ ("armv6l", "armv7l", "i686"), platforms)
```

Comment:
Too many Rust packages do this, but they usually force the base architecture (so why doing that to start with?  no clue), if that's the case you can use `lock_microarchitecture=false` to skip the check